### PR TITLE
Added runtime context in current.

### PR DESCRIPTION
**Pull request by valayDave**

### CODE REVIEW

File: metaflow/plugins/aws/batch/batch_decorator.py [hunk 1]
```diff
@@ -206,6 +206,7 @@ def task_pre_step(
     ):
         self.metadata = metadata
         self.task_datastore = task_datastore
+        current._update_env({"runtime_context": "batch"})
```
Comment (savingoyal) [inline metaflow/plugins/aws/batch/batch_decorator.py:209]: `aws-batch`

File: metaflow/plugins/kubernetes/kubernetes_decorator.py [hunk 2]
```diff
@@ -267,6 +268,7 @@ def task_pre_step(
     ):
         self.metadata = metadata
         self.task_datastore = task_datastore
+        current._update_env({"runtime_context":"kubernetes"})
```
Comment (savingoyal) [inline metaflow/plugins/kubernetes/kubernetes_decorator.py:271]: This hook is not guaranteed to always execute on Kubernetes.

File: metaflow/plugins/kubernetes/kubernetes_decorator.py [hunk 3]
```diff
@@ -3,6 +3,7 @@
 import sys
 
 import requests
+from metaflow import current
```
Comment (savingoyal) [inline metaflow/plugins/kubernetes/kubernetes_decorator.py:6]: Can you also handle the scenario when the context is local?

**Comment by valayDave** [comment]
Can we merge this ?

**Pull request closed without merging**